app/models/container.rb

```diff
@@ -22,7 +22,7 @@ class Container < ApplicationRecord
     format: { with: HOSTNAME_FORMAT },
```

Comment:
should put examples on readme or some docs on what to put as value for image_server and image_procotol